run_in_sequence.py

```diff
@@ -40,36 +43,31 @@ def parse_arguments():
 
     args = parser.parse_args()
 
+    # Handle Tag ANDing properly
+    args.tags = create_behave_tags(args.test_tags)
+
     return args
 
 
 def main():
     """
     Runner
     """
-
     if not is_valid_sequential_environment():
-        logger.error(
-            "Error: Environment Variable(s) must be set as 'IGNORE_NON_STANDALONE_DATA_SETUP=False'")
+        logger.error("Error: Environment Variable(s) must be set as 'IGNORE_SEQUENTIAL_DATA_SETUP=False'")
 
     args = parse_arguments()
 
-    logger.error('command_line_args=' + args.command_line_args)
-    logger.error('args.format=' + args.format)
-    logger.error('args.tags=' + args.tags)
-    logger.error('args.system_features_directory' + args.system_features_directory)
-    logger.error('args.acceptance_features_directory' + args.acceptance_features_directory)
-
-    behave_executable.main(
-        args=f'--stop {args.command_line_args} '
-             f'--format {args.format} '
-             f'--tags={args.tags} '
-             f'{args.system_features_directory}')
-
-    behave_executable.main(
-        args=f'--stop {args.command_line_args} '
-             f'--format {args.format}'
-             f' --tags={args.tags} {args.acceptance_features_directory}')
+    behave_args = f'{args.command_line_args} --format {args.format} {args.tags} {args.system_features_directory}'
+    exitCode = behave_executable.main(behave_args)
+
+    if exitCode != 0:
+        sys.exit(exitCode)
+
+    behave_args = f'{args.command_line_args} --format {args.format} {args.tags} {args.acceptance_features_directory}'
+    exitCode = behave_executable.main(behave_args)
+
+    sys.exit(exitCode)
```

Comment:
👍 

You can just return a nonzero integer from a main function, but this will work